untrusted/src/lib.rs

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) 2022 The MobileCoin Foundation
```

Comment:
The first crate should handle the linkage and bindings and export the API (i.e. a pure FFI crate).

The *second* crate would be the one that has the ideomatic rust types (this also means you can nuke various "allow(thing_that_looks_like_c)" stuff.

Reply:
This should be addressed now with the separation into 3 crates